api/src/jvmMain/kotlin/com/buenaflor/kflogger/KLogContext.jvm.kt

```diff
@@ -0,0 +1,7 @@
+package com.buenaflor.kflogger
+
+@Suppress("NO_ACTUAL_CLASS_MEMBER_FOR_EXPECTED_CLASS")
+public actual typealias KLogContext<LOGGER, API> = LogContext<LOGGER, API>
+
+@Suppress("NO_ACTUAL_CLASS_MEMBER_FOR_EXPECTED_CLASS")
```

Comment:
What error is this suppressing? Which member is missing? Could you include a comment about why this needs to be suppressed as well

Reply:
The first suppress is for 'errors' regarding the protected visibility: https://youtrack.jetbrains.com/issue/KT-19848
The second suppress is for companion objects.

Both will work if you suppress them